drivers/jdbc-driver/api/jdbc-driver.api

```diff
@@ -63,6 +63,7 @@ public final class app/cash/sqldelight/driver/jdbc/JdbcPreparedStatement : app/c
 	public final fun bindInt (ILjava/lang/Integer;)V
 	public fun bindLong (ILjava/lang/Long;)V
 	public final fun bindObject (ILjava/lang/Object;)V
+	public final fun bindObjectOther (ILjava/lang/Object;)V
```

Comment:
Not yet released 🎉

Reply:
Cool. Wish this plugin did actual validation like Metalava so it was clear in your PR the rename was safe.

Reply:
Metalava is Androids api validator, isn't it? Is it also able to do validate synthetic changes, like in 00ae6214?